be/src/vec/columns/predicate_column.h

```diff
@@ -134,17 +134,15 @@ class PredicateColumnType final : public COWHelper<IColumn, PredicateColumnType<
     }
 
     void insert_many_default_type(const char* data_ptr, size_t num) {
-        T* res_val_ptr = (T*)data.get_end_ptr();
-        memcpy(res_val_ptr, data_ptr, num * sizeof(T));
-        res_val_ptr += num;
-        data.set_end_ptr(res_val_ptr);
```

Comment:
If not change end ptr, then the podarray's data size is not changed? Is it wrong?

Reply:
`data.resize(old_size + num);` will alloc memory and change end ptr.